channeldb/mp_payment.go

```diff
@@ -0,0 +1,137 @@
+package channeldb
+
+import (
+	"time"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/lightningnetwork/lnd/lntypes"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/routing/route"
+)
+
+// MPPaymentCreationInfo is the information necessary to have ready when
+// initiating a payment, moving it into state InFlight.
+type MPPaymentCreationInfo struct {
+	// PaymentHash is the hash this payment is paying to.
+	PaymentHash lntypes.Hash
+
+	// PaymentAddr is a receiver-generated nonce used to prevent concurrent
+	// payments from colliding.
+	PaymentAddr lntypes.Hash
+
+	// Value is the amount we are paying.
+	Value lnwire.MilliSatoshi
+
+	// CreatingDate is the at which this payment was started.
+	CreationDate time.Time
```

Comment:
`CreationTime` ?

Reply:
Also added a `creation_time_ns` to the rpc response and deprecated `creation_date`